microsoft-edge/devtools-guide-chromium/memory-problems/index.md

```diff
@@ -281,18 +276,186 @@ DevTools now tracks all of the objects that were GC'd during the recording.  Use
 
 If you are investigating objects that were only GC'd during specific major or minor GC operations, configure the settings appropriately to track the operation you care about. To learn more about the differences between major and minor GC, see [Trash talk: the Orinoco garbage collector | V8 JavaScript engine developer blog](https://v8.dev/blog/trash-talk).
 
+
+<!-- ====================================================================== -->
+## Find DOM tree memory leaks from detached elements
+
+A DOM node is only garbage-collected when there are no references to the node from either the DOM tree or JavaScript code running on the page.  A node is said to be "detached" when it is removed from the DOM tree but some JavaScript still references it.  Detached DOM nodes are a common cause of memory leaks.
+
+
+<!-- ------------------------------ -->
+#### Tools for investigating detached elements
+
+| Tool | Pros | Cons | Docs |
+|---|---|---|---|
+| **Memory** tool > **Detached elements** option button (profiling type) | Shows elements as DOM tree nodes. | Doesn't link to JavaScript source code. | [Find DOM tree memory leaks ("Detached elements" profiling type)](#find-dom-tree-memory-leaks-detached-elements-profiling-type), below |
+| **Memory** tool > **Heap snapshot** option button (profiling type) | The detached element has a link to its JavaScript source code.<!-- todo: how get to JS? --> | Shows elements as objects in memory. | [Find DOM tree memory leaks ("Heap snapshot" profiling type > Detached)](#find-dom-tree-memory-leaks-heap-snapshot-profiling-type--detached), below |
+| **Detached Elements** tool | Shows elements as DOM tree nodes.  The detached element has a link to its JavaScript source code. | Deprecated. | [Debug DOM memory leaks by using the Detached Elements tool](./dom-leaks.md) |
+
+
+<!--[4]-->
+<!-- ====================================================================== -->
+## Find DOM tree memory leaks ("Detached elements" profiling type)
```

Comment:
Create an article and move the content from this h2 section into that article.  Keep this h2 section, but in it, just link to the new article.

The present article is too long for LMC, which is designed for short articles.  When we move the explanatory technical content from the DE tool article into this h2 section, this section will be too long.  This section needs to just link out to a dedicated article, like the "Heap snapshot" option button / profiling type h2 section 
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/memory-problems/?branch=pr-en-us-3282#record-a-heap-snapshot-heap-snapshot-profiling-type
links out to the Record Heap Snapshots article:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshots?branch=pr-en-us-3282

Either:
* Create that article in a future PR, when DE tool is removed and DE tool article is removed.
* In the present PR, build-out this article's existing design means creating a dedicated article for DE profiling type now, just like the dedicated article for Heap snapshot profiling type.

Reply:
created article, moved Mem tool > DE section content to there and linked to there, copied/trimmed explanatory content from DE tool article with Maint comments